packages/firebase_ml/lib/firebase_ml.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library firebase_ml;
+
+import 'dart:async';
+import 'dart:io';
+import 'package:flutter/services.dart';
+
+part 'src/firebase_ml.dart';
```

Comment:
We've started discouraging this type of pattern where a library is split into separate files using `part` and `part of`. It tends to make it more difficult to read through code. The line `part of library_name` is now only available for legacy reasons.

I would recommend removing the imports from this file and to export the library files instead. e.g.
```
library firebase_ml;

export 'src/firebase_ml.dart';
export 'src/firebase_model_manager.dart';
export 'src/firebase_model_download_conditions.dart';
export 'src/firebase_remote_model.dart';
```

I would also take a look at [Effective Dart on libraries](https://dart.dev/guides/language/effective-dart/usage#libraries).

Reply:
Done, thanks for the guide